# Allow configure to succeed with recent versions of libtasn1
# Gentoo: https://bugs.gentoo.org/show_bug.cgi?id=266554
# Gnome: https://bugzilla.gnome.org/show_bug.cgi?id=560869
--- a/configure.in	2008-06-30 17:29:13.000000000 +0200
+++ b/configure.in	2009-04-18 12:16:56.000000000 +0200
@@ -381,17 +381,19 @@
 
 GKR_LIBTASN1_VERSION=0.3.4
 
-AM_PATH_LIBTASN1($GKR_LIBTASN1_VERSION,,
-	AC_MSG_ERROR([[
-*** 
-*** LibtASN1 ${GKR_LIBTASN1_VERSION} was not found. 
-]]))
-libtasn=yes
-AC_DEFINE_UNQUOTED(GKR_LIBGCRYPT_VERSION, "$GKR_LIBTASN1_VERSION", [Version of libtasn1 we expect])
+PKG_CHECK_MODULES(LIBTASN1,libtasn1 >= 1.0)
+AC_SUBST([LIBTASN1_LIBS])
+AC_SUBST([LIBTASN1_CFLAGS])
 
 DAEMON_CFLAGS="$DAEMON_CFLAGS $LIBTASN1_CFLAGS"
 DAEMON_LIBS="$DAEMON_LIBS $LIBTASN1_LIBS"
 
+AC_PATH_PROG(ASN1PARSER, asn1Parser, no)
+if test "$ASN1PARSER" = "no" ; then
+	AC_MSG_ERROR(asn1Parser tool is not installed)
+fi
+
+
 dnl ==========================================================================
 
 dnl Turn on the additional warnings last, so -Werror doesn't affect other tests.
